### PR TITLE
Create zip archives

### PR DESCRIPTION
**Pull request by hikhvar**
zip archives are easier accessible for Windows users

### CODE REVIEW

File: create_package.sh [hunk 1]
```diff
@@ -11,4 +11,5 @@ do
   FILENAME="$(basename "$f")"
   echo "Create pacakge for ${FILENAME}"
   tar cfz "${DEST}/${FILENAME}.tar.gz" "./keys/${FILENAME}" "./ips/${FILENAME}"
+  zip "${DEST}/${FILENAME}.zip" "./keys/${FILENAME}" "./ips/${FILENAME}"
```
Comment (johscheuer) [inline create_package.sh]: I dislike this, I don't see any real value creating tar and zip archives either we only use zip or tar.

Comment (hikhvar) [reply create_package.sh]: I switched to zip archives.

**Comment by johscheuer** [comment]
Seems like you need to rebase your changes

**Comment by hikhvar** [comment]
Rebased.

**Pull request merged**